### ISSUE TITLE
High CPU usage when running tflint

### ISSUE THREAD

**Issue by frezbo** [wata727/tflint#72]
When running tflint for the first time there is a high CPU spike after some time and system gets stuck for a while
OS:
```
Fedora 24 4.9.13-201.fc25.x86_64
```
Using https://github.com/wata727/tflint/releases/download/v0.3.1/tflint_linux_amd64.zip this binary

**Comment by wata727** [comment]
Probably, the cause of this problem is `aws_instance_invalid_ami` detector. This detector call `DescribeImages` internally. The result includes a large amount of available AMIs.
However this detector is useful. So please use the `--fast` option if CPU becomes a bottleneck. When this option is enabled, Ignore slow detectors.

**Comment by frezbo** [comment]
@wata727 when if run tflint with the debug option, I don't see the high CPU usage
```
tflint --debug
```
finished fast compared to: 
```
tflint
```

**Comment by frezbo** [comment]
@wata727 you may close this issue if you think it is not relevant, but I still don't understand my whole system becoming stuck for 10-20 seconds
